go.work.sum

```diff
@@ -1,61 +1,102 @@
-cloud.google.com/go v0.26.0 h1:e0WKqKTd5BnrG8aKH3J3h+QvEIQtSUcf2n5UZ5ZgLtQ=
-github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
 github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46 h1:lsxEuwrXEAokXB9qhlbKWPpo3KMLZQ5WB5WLQRW1uq0=
-github.com/PuerkitoBio/purell v1.0.0 h1:0GoNN3taZV6QI81IXgCbxMyEaJDXMSIjArYBCYzVVvs=
-github.com/PuerkitoBio/urlesc v0.0.0-20160726150825-5bd2802263f2 h1:JCHLVE3B+kJde7bIEo5N4J+ZbLhp0J1Fs+ulyRws4gE=
-github.com/census-instrumentation/opencensus-proto v0.2.1 h1:glEXhBS5PSLLv4IXzLA5yPRVX4bilULVyxxbrfOtDAk=
-github.com/client9/misspell v0.3.4 h1:ta993UF76GwbvJcIo3Y68y/M3WxlpEHPWIGDkJYwzJI=
+github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46/go.mod h1:3wb06e3pkSAbeQ52E9H9iFoQsEEwGN64994WTCIhntQ=
```

Comment:
It doesn't make sense to look at `go.work.sum`, IMHO. It's Go machinery, and we can't control that. Also, its purpose is unclear (see: https://stackoverflow.com/a/71613130/844449). Furthermore, [go workspaces had a major overhaul in go1.22](https://tip.golang.org/doc/go1.22#go-command), so I wouldn't bother with this.

The changes here are because of changes to `go.mod` files:

- [`go.mod`](https://github.com/knative/hack/pull/386/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6) - just Go version bump
- [`schema/go.mod`](https://github.com/knative/hack/pull/386/files#diff-178b6fad923a512b840e79c4bdfe41af41b023666ce492bce1cac4163dc7f396) - a bump of Go and K8s API versions
- [`test/go.mod`](https://github.com/knative/hack/pull/386/files#diff-25e9c6800a800713f0b8f25b203c4f9186120896512a0f9f0dee725e372a5b17) - a Go version bump, add of K8s code-gen, add of charmbracelet/gum, and a couple of minor test things
- [`test/vendorproj/go.mod`](https://github.com/knative/hack/pull/386/files#diff-ac94ad59adeee108038293f3877a1f4933d06a25c8ba2546dd09e9af6d184df0) - Go version bump, testify patch bump

Remember, that only the main `go.mod` file is consumed by Knative projects, so changes to other submodules are not affecting downstream projects.